featuresScore.java

```diff
@@ -0,0 +1,76 @@
+
+public class featuresScore {
+	public static final int COLS = 10;
+	public static final int ROWS = 21;
```

Comment:
Oh and please check for the regressions due to changing the `ROWS` to 20 (like the loop conditions and stuff)